tests/ballerina-integration-test/src/test/resources/http/httpservices/01_100_continue.bal

```diff
@@ -16,30 +16,30 @@ service<http:Service> helloContinue bind { port: 9090 } {
     hello(endpoint caller, http:Request request) {
         if (request.expects100Continue()) {
             log:printInfo("Sending 100-Continue response");
-            caller->continue() but {
-                error e => log:printError("Error sending response", err = e)
-            };
+            var responseError = caller-> continue();
+            if (responseError is error) {
+                log:printError("Error sending response", err = responseError);
+            }
             log:printInfo("100-Continue response sent");
         }
 
         http:Response res = new;
+        var result  = request.getTextPayload();
 
-        match request.getTextPayload() {
-            string payload => {
-                res.statusCode = 200;
-                res.setPayload(untaint payload);
-                caller->respond(res) but {
-                    error e => log:printError("Error sending response", err = e)
-                };
+        if (result is string) {
+            res.statusCode = 200;
```

Comment:
Isn't these two lines are redundant?? We can simply pass the payload for the respond.